mne/decoding/tests/test_time_gen.py

```diff
@@ -70,7 +70,15 @@ def test_generalization_across_time():
     assert_equal("<GAT | fitted, start : -0.200 (s), stop : 0.499 (s), no "
                  "prediction, no score>", '%s' % gat)
     assert_equal(gat.ch_names, epochs.ch_names)
+    # test different predict function:
+    gat.decision_function(epochs)
+    assert_array_equal(np.shape(gat.y_pred_), (15, 15, 14, 1))
+    gat.predict_proba(epochs)
+    assert_array_equal(np.shape(gat.y_pred_), (15, 15, 14, 2))
```

Comment:
In the sklearn examples I can find, predicted probabilities aren't stored in the object, but in an external variable. I think it would here be more natural to store them in the gat object though.